src/ClientData/include/ClientData/ThreadTrackDataProvider.h

```diff
@@ -50,7 +50,10 @@ class ThreadTrackDataProvider final {
   [[nodiscard]] std::vector<const orbit_client_protos::TimerInfo*> GetTimers(
       uint32_t thread_id, uint64_t min_tick = std::numeric_limits<uint64_t>::min(),
       uint64_t max_tick = std::numeric_limits<uint64_t>::max(), bool exclusive = false) const {
-    return GetScopeTreeTimerData(thread_id)->GetTimers(min_tick, max_tick, exclusive);
+    const auto* scope_tree_timer_data = GetScopeTreeTimerData(thread_id);
+    return scope_tree_timer_data == nullptr
+               ? std::vector<const orbit_client_protos::TimerInfo*>{}
+               : scope_tree_timer_data->GetTimers(min_tick, max_tick, exclusive);
```

Comment:
Nit and pers pref: I would return early and avoid writing the entire type in the nullptr case:

```cpp
if (scope_tree_timer_data == nullptr) return {};
return scope_tree_timer_data->GetTimers(min_tick, max_tick, exclusive);
```

Reply:
done